src/libraries/System.Collections/src/System/Collections/Generic/OrderedDictionary.cs

```diff
@@ -6,12 +6,8 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-#if NET8_0_OR_GREATER
 using static System.ArgumentNullException;
 using static System.ArgumentOutOfRangeException;
```

Comment:
```suggestion
 #if NET
 using static System.ArgumentNullException;
 using static System.ArgumentOutOfRangeException;
 #else
 using static System.Collections.ThrowHelper;
 #endif
```

This file is compiled for `netstandard2.0` in `System.Text.Json`.

https://github.com/dotnet/runtime/blob/main/src/libraries/System.Text.Json/src/System.Text.Json.csproj#L366

Reply:
Thank you. I will be sorting out the build failures.